packages/e2e-tests/src/specs/editor/floatingMenu.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+    createNewStory,
+    waitForCarousel
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Floating Menu', () => {
+    beforeEach(async () => {
+        await createNewStory();
+    });
+
+    it('should display text floating menu', async () => {
+        // Open a text tab
+        await expect(page).toClick('#library-tab-text');
+
+        // Add a paragraph
+        await expect(page).toClick('button[data-testid="preview-text"] span', {
```

Comment:
`#library-pane-text button` should also work? without having the need to add testid.

Reply:
As far as I know it's good to use only test ids for tests, because for example ids can be changed and tests will fail.

Reply:
@merapi I've used `#library-pane-text button` like you suggested

Reply:
Ok, testids are not bad, but we don't plan to change ids or labels often (forever if possible) and a lot of tests can fail if we change any of them.